assets/source/catalog-sync/sections/SyncState.js

```diff
@@ -30,6 +34,28 @@ const SyncState = () => {
 			</CardHeader>
 			<SyncStateSummary overview={ feedState?.overview } />
 			<SyncStateTable workflow={ feedState?.workflow } />
+			<CardFooter justify="flex-start">
+				<Icon icon={ trendingUpIcon } />
+				<Text>
+					{ createInterpolateElement(
+						__(
+							'Set up and manage ads to increase your reach with <adsManagerLink>Pinterest ads manager</adsManagerLink>',
+							'pinterest-for-woocommerce'
+						),
+						{
+							adsManagerLink: (
+								<ExternalLink
+									className="link"
```

Comment:
```suggestion
```

In case there is no specific reason to use a custom selector. I would go for the default.